todo_app/accounts/api.py

```diff
@@ -0,0 +1,60 @@
+from rest_framework import generics, permissions
+from rest_framework.response import Response
+from knox.models import AuthToken
+from .serializers import UserSerializer, RegisterSerializer, LoginSerializer
+from django.contrib.auth.models import User
+
+# Register API
```

Comment:
No need for a comment, here. It's the same thing as the name of the class.
Also, too many blank lines. There should just be two lines separating each class from the things around it.